test/rosenbrock4.cpp

```diff
@@ -127,6 +127,24 @@ BOOST_AUTO_TEST_CASE( test_rosenbrock4_dense_output )
     stepper.calc_state( 0.5 * ( tr.first + tr.second ) , x );
 }
 
+BOOST_AUTO_TEST_CASE( test_rosenbrock4_dense_output_ref )
+{
+    typedef rosenbrock4_dense_output< boost::reference_wrapper< rosenbrock4_controller< rosenbrock4< value_type > > > > stepper_type;
+    typedef rosenbrock4_controller< rosenbrock4< value_type > > controlled_stepper_type;
+    controlled_stepper_type  c_stepper;
+    stepper_type stepper( boost::ref( c_stepper ) );
```

Comment:
Is there a way to verify that indeed a reference to c_stepper is used, e.g by changing the error control thresholds of c_stepper and check if `stepper` responds to such changes in `c_stepper`?

Reply:
Yes, I will add this.